theano/tensor/tests/test_extra_ops.py

```diff
@@ -337,6 +337,57 @@ def test_var_interface(self):
             assert numpy.allclose(tested, expected)
 
 
+class CompressTester(utt.InferShapeTester):
+    axis_list = [None,
+                 -1,
+                 0,
+                 0,
+                 0,
+                 1]
+    cond_list = [[1, 0, 1, 0, 0, 1],
+                 [0, 1, 1, 0],
+                 [0, 1, 1, 0],
+                 [],
+                 [0, 0, 0, 0],
+                 [1, 1, 0, 1, 0]]
+    shape_list = [(2, 3),
+                  (4, 3),
+                  (4, 3),
+                  (4, 3),
+                  (4, 3),
+                  (3, 5)]
+
+    def setUp(self):
+        super(CompressTester, self).setUp()
+        self.op = compress
+
+    def test_op(self):
+        for axis, cond, shape in zip(self.axis_list, self.cond_list, self.shape_list):
+            cond_var = theano.tensor.ivector()
+            data     = numpy.random.random(size=shape).astype(theano.config.floatX)
+            data_var = theano.tensor.matrix()
+
+            f = theano.function([cond_var, data_var], self.op(cond_var, data_var, axis=axis))
+
+            expected = numpy.compress(cond, data, axis=axis)
+            tested = f(cond, data)
+
+            assert tested.shape == expected.shape
+            assert numpy.allclose(tested, expected)
+
+    def test_infer_shape(self):
```

Comment:
As this isn't an op anymore, but a graph of existing op, there is not need to test infer_shape in particular. Can you just remove this test?